{
  "Header": {
    "Original": "CVE-2022-25304 (All versions of package opcua; all versions of package asyncua are vul ...)",
    "Line": 162822,
    "ID": "CVE-2022-25304",
    "Description": "(All versions of package opcua; all versions of package asyncua are vul ...)"
  },
  "Annotations": [
    {
      "Original": "- python-opcua \u003cremoved\u003e (bug #1021274)",
      "Line": 162823,
      "Type": "package",
      "Package": "python-opcua",
      "Kind": "removed",
      "BugNo": 1021274
    },
    {
      "Original": "[bullseye] - python-opcua \u003cno-dsa\u003e (Minor issue)",
      "Line": 162824,
      "Type": "package",
      "Release": "bullseye",
      "Package": "python-opcua",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "[buster] - python-opcua \u003cno-dsa\u003e (Minor issue)",
      "Line": 162825,
      "Type": "package",
      "Release": "buster",
      "Package": "python-opcua",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "NOTE: https://github.com/FreeOpcUa/python-opcua/issues/1466",
      "Line": 162826,
      "Type": "NOTE",
      "Description": "https://github.com/FreeOpcUa/python-opcua/issues/1466"
    },
    {
      "Original": "NOTE: https://security.snyk.io/vuln/SNYK-PYTHON-OPCUA-2988730",
      "Line": 162827,
      "Type": "NOTE",
      "Description": "https://security.snyk.io/vuln/SNYK-PYTHON-OPCUA-2988730"
    }
  ]
}